### PR TITLE
Update dendron.topic.graph-view.md

### PR DESCRIPTION
**Pull request by ens100**
Correcting a couple of typos and addition of key strokes.

### CODE REVIEW

File: vault/dendron.topic.graph-view.md [hunk 1]
```diff
@@ -11,23 +11,26 @@ custom: {}
...
+Instead of showing edges as links between nodes, Dendron edges are the hierarchical relationships between your notes. Think of it as a 20,000 foot view of the structure of your knowledge base. 
 
-Note that this command is not optimized for performance and you might notice slowdowns if you have more than a few hundred notes. If you are running into performance issues, please upvote this [issue](https://github.com/dendronhq/dendron/issues/107) to prioritize the performacne work!
+Note that this command is not optimized for performance and you might notice slowdowns if you have more than a few hundred notes. If you are running into performance issues, please upvote this [issue](https://github.com/dendronhq/dendron/issues/107) to prioritize the performance work!
 
 ## Commands
 
 ### Dendron: Show Note Graph
 
+Ctrl + Shift + P: `> Dendron: Show Note Graph`
```
Comment (kevinslin) [inline vault/dendron.topic.graph-view.md:24]: this is different depending on os. i would say 'launch the command bar' with a link to [this](https://www.dendron.so/notes/401c5889-20ae-4b3a-8468-269def4b4865.html#launch-the-command-bar)

File: vault/dendron.topic.graph-view.md [hunk 2]
```diff
@@ -11,23 +11,26 @@ custom: {}
...
+
 Show your note hierarchies visually in a graph.
 
 ![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/graph.show-notes.gif)
 
-
 ### Dendron: Show Schema Graph
 
+Ctrl + Shift + P: `> Dendron: Show Schema Graph`
```
Comment (kevinslin) [inline vault/dendron.topic.graph-view.md:32]: same comment as above

**Review by kevinslin — CHANGES_REQUESTED** [review]
content is good, can you update the reference for keyboard shortcuts so its not windows specific?

**Comment by ens100** [comment]
I think I have submitted a revised file based on your comments but not sure if I did this correctly (still trying to get my head around GitHub)

**Comment by kevinslin** [comment]
all good. just merged the changes. thanks for the corrections 👍

**Pull request merged**